README.md

```diff
@@ -1,2 +1,44 @@
-# aerogear-web-sdk
+# AeroGear Web SDK
+[![License](https://img.shields.io/badge/-Apache%202.0-blue.svg)](https://opensource.org/s/Apache-2.0)
```

Comment:
Can you add a circle ci build status shield?

Reply:
Good point! Done